src/OSPSuite.Infrastructure.Import/Services/Importer.cs

```diff
@@ -56,15 +58,17 @@ public class Importer : IImporter
       private readonly IDataSetToDataRepositoryMapper _dataRepositoryMapper;
       private readonly IDimension _molWeightDimension;
 
-      public Importer(IoC container, IDataSourceFileParser parser, IDataSetToDataRepositoryMapper dataRepositoryMapper, IDimensionFactory dimensionFactory)
+      public Importer(IoC container, IDataSourceFileParser parser, IDataSetToDataRepositoryMapper dataRepositoryMapper,
```

Comment:
Let's align all the parameters in the constructor sonthat we habe one by line

Reply:
done